src/content/operations/known_issues/_index.en.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "Known Issues"
+date = 2020-08-12T16:02:00+02:00
+weight = 50
++++
+
+## General
+
+* Minimum supported Kubernetes version is 1.17.
+* Submariner only supports kube-proxy in iptables mode. IPVS is not supported at this time.
+* Clusters deployed with the Calico network plug-in require further configuration to be compatible with Submariner. Please refer to the
+[Calico-specific deployment instructions](../deployment/calico/).
+
```

Comment:
Only CoreDNS is supported out of the box, in KubeDNS the DNS stub domain needs to be added manually, see the GKE docs

Reply:
Good point, added